code/modules/clothing/clothing.dm

```diff
@@ -453,6 +474,40 @@ BLIND     // can't see anything
 	if(!lavaland_equipment_pressure_check(T))
 		. *= high_pressure_multiplier
 
+/obj/item/clothing/emp_act(severity)
+	if(tracking_bug)
+		tracking_bug.emp_act(severity)
+	. = ..()
```

Comment:
Register a signal instead on the tracking bug when you attach it to some clothing